### PR TITLE
Release k6 v0.33.0

### PR DESCRIPTION
**Pull request by na--**

### CODE REVIEW

File: lib/consts/consts.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ import (
 )
 
 // Version contains the current semantic version of k6.
-const Version = "0.32.0"
+const Version = "0.33.0"
```
Comment (imiric) [inline lib/consts/consts.go:31]: Let's split this into a separate commit.

Comment (na--) [reply lib/consts/consts.go:31]: :+1: will do at the end, after addressing any other feedback, to save myself some effort when rebasing :sweat_smile:

**Review by MStoykov — APPROVED** [review]

**Review by imiric — APPROVED** [review]
Doesn't seem like you missed anything :+1: 

Let's just split the commit before merging.

**Review by MStoykov — APPROVED** [review]

**Review by imiric — APPROVED** [review]
